modules/native/src/lib.rs

```diff
@@ -1,12 +1,11 @@
-#[macro_use]
-extern crate neon;
-
 use neon::prelude::*;
+use neon::{class_definition, declare_types, impl_managed, register_module};
 
 fn hello(mut cx: FunctionContext) -> JsResult<JsString> {
     Ok(cx.string("hello node"))
 }
 
+#[allow(dead_code)]
```

Comment:
I don't understand why this is dead code.

Reply:
Rust complains about fields of User struct is never used.

Reply:
Oh, the _fields_!